### DISCUSSION TITLE
Is it possible to upgrade from 1.9.0 to 1.10.0 remotely?

### DISCUSSION THREAD

**Discussion by VeniVidiViciLA** [raspiblitz/raspiblitz#4245] [Q&A] [answered]
Hi guys,

I have a node running on 1.9.0 on the other side of the planet and I would like to know if there is a good and safe way to upgrade it to 1.10.0 by any chance using SSH ?

Thank you all

**Comment by rootzoll** [comment, marked as answer]
nope - major & minor updates (like from 1.9 to 1.10) always need a fresh sd card image. Only patch updates (like from 1.9.0 to 1.9.1) can be done remotely with commans `patch` (or SSH-MENU: UPDATE > PATCH)

**Reply by VeniVidiViciLA** [reply]
Thank you @rootzoll 
One last question, would it be possible to only update LND from 0.16.2-beta to 0.17.x maybe ? I saw couple of LND scripts here https://github.com/openoms/lightning-node-management/blob/master/lnd.updates/lnd.update.v0.13.3-beta.sh. but those are outdated. Could I use those as a base ?

Thank you again guys !

**Comment by rootzoll** [comment]
In the SSH-MENU there is the "UPDATE > LND > RECKLESS" option to update LND to the latest release. But that is exactly how its named - reckless and can have side effects. I would not recommend to do this on a v1.9.0 version.
